src/Compilers/CSharp/Test/Semantic/Semantics/ReadOnlyStructsTests.cs

```diff
@@ -1444,6 +1444,35 @@ public struct S
                 Diagnostic(ErrorCode.ERR_InvalidPropertyReadOnlyMods, "set").WithArguments("S.P8").WithLocation(11, 44));
         }
 
+        [Fact]
+        public void NetModule_ImplicitReadOnlyAutoProperty()
+        {
+            var csharp = @"
+public struct S
+{
+    public int P1 { get; private set; }
+}
+";
+            var comp = CreateCompilation(csharp, options: TestOptions.DebugModule, targetFramework: TargetFramework.Mscorlib45);
+            comp.VerifyDiagnostics();
```

Comment:
Should verify the member does not have the `IsReadOnlyAttribute` on it. Makes it explicit for future developers what you're testing for here .

Reply:
Done